.github/workflows/publish-packages.yml

```diff
@@ -323,5 +307,136 @@ jobs:
 
       - name: Uninstall Hazelcast from homebrew
         run: |
-          source ./common.sh          
+          source ./common.sh
           brew uninstall ${{ env.HZ_DISTRIBUTION}}@$BREW_PACKAGE_VERSION
+
+  buildVagrant:
+    runs-on: macos-12
+    needs: [ prepare, rpm, deb ]
```

Comment:
Do you need `prepare` here?

Reply:
For the `HZ_VERSION: ${{ needs.prepare.outputs.hz_version }}`